packages/autometrics-lib/tests/objectives.test.ts

```diff
@@ -0,0 +1,101 @@
+import {
+  PeriodicExportingMetricReader,
+  InMemoryMetricExporter,
+} from "@opentelemetry/sdk-metrics";
+import { afterEach, beforeAll, describe, expect, test } from "vitest";
+import {
+  autometrics,
+  init,
+  ObjectiveLatency,
+  ObjectivePercentile,
+} from "../src";
+import { getMetricsProvider } from "../src/instrumentation";
+import { collectAndSerialize } from "./util";
+
+let exporter: PeriodicExportingMetricReader;
+
+describe("Autometrics objectives test", () => {
+  beforeAll(async () => {
+    exporter = new PeriodicExportingMetricReader({
+      exporter: new InMemoryMetricExporter(0),
```

Comment:
Can you maybe assign this to a variable or maybe add a comment as to what `0` is meant for/to do?